src/main/kotlin/com/tally/kiosk/shop/persistence/MenuJpaRepository.kt

```
@@ -0,0 +1,7 @@
+package com.tally.kiosk.shop.persistence
+
+import com.tally.kiosk.shop.entity.Menu
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface MenuJpaRepository : JpaRepository<Menu, Long> {
+}
```

detekt (1.23.7)

[warning] The class or object MenuJpaRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)